extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/internal/WeatherUndergroundHandlerFactory.java

```diff
@@ -59,7 +81,8 @@ protected void unsetUnitProvider(final UnitProvider unitProvider) {
         this.unitProvider = null;
     }
 
-    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_WEATHER);
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Sets.union(BRIDGE_THING_TYPES_UIDS,
```

Comment:
to avoid guava please use something like 

```
private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Stream
            .of(BRIDGE_THING_TYPES_UIDS, WeatherUndergroundBindingConstants.SUPPORTED_THING_TYPES_UIDS)
            .flatMap(x -> x.stream()).collect(Collectors.toSet());
```

Its not as elegant, maybe you can find even another way...

Reply:
This looks actually pretty cool to me ;-) changed.